### PR TITLE
Pp 387 add onboarding screen events

### PR DESCRIPTION
**Pull request by ValentinaIancu-Gini**
Add missing `get_started_tapped ` and `skip_tapped` events from the bottom navigation bar.
Add missing `custom_onboarding_title ` property in all events for custom onboarding screens.

### CODE REVIEW

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingViewController.swift [hunk 1]
```diff
@@ -149,21 +148,34 @@ class OnboardingViewController: UIViewController {
     }
 
     @objc private func nextPage() {
-        let currentPageScreenName = dataSource.pageModels[dataSource.currentPageIndex].analyticsScreen
```
Comment (mrkulik) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingViewController.swift:151]: [Suggestion] extra line not needed

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingViewController.swift [hunk 2]
```diff
@@ -149,21 +148,34 @@ class OnboardingViewController: UIViewController {
...
-            AnalyticsManager.track(event: .getStartedTapped, screenNameString: currentPageScreenName)
-            close()
+           getStartedButtonAction()
         }
     }
 
+    private func getStartedButtonAction() {
+        track(event: .getStartedTapped, for: dataSource.currentPageIndex)
+        close()
+    }
```
Comment (mrkulik) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingViewController.swift:166]: [Suggestion] missing line spacing

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingViewController.swift [hunk 3]
```diff
@@ -149,21 +148,34 @@ class OnboardingViewController: UIViewController {
     }
 
     @objc private func nextPage() {
-        let currentPageScreenName = dataSource.pageModels[dataSource.currentPageIndex].analyticsScreen
 
         if dataSource.currentPageIndex < dataSource.pageModels.count - 1 {
             // Next button tapped
-            AnalyticsManager.track(event: .nextStepTapped, screenNameString: currentPageScreenName)
+            track(event: .nextStepTapped, for: dataSource.currentPageIndex)
```
Comment (mrkulik) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingViewController.swift:154]: [Comment] `dataSource.currentPageIndex` is used 3 times in a row, probably we can declare it 1 time and reuse

File: CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingViewController.swift [hunk 4]
```diff
@@ -149,21 +148,34 @@ class OnboardingViewController: UIViewController {
...
             let index = IndexPath(item: dataSource.currentPageIndex + 1, section: 0)
             pagesCollection.scrollToItem(at: index, at: .centeredHorizontally, animated: true)
             dataSource.isProgrammaticScroll = true
         } else {
-            // Get started button tapped
-            AnalyticsManager.track(event: .getStartedTapped, screenNameString: currentPageScreenName)
-            close()
+           getStartedButtonAction()
```
Comment (mrkulik) [inline CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/OnboardingViewController.swift:159]: [Suggestion] please check indentation

**Review by mrkulik — CHANGES_REQUESTED** [review]
@ValentinaIancu-Gini great job! few improvements suggested

**Comment by ValentinaIancu-Gini** [comment]
@mrkulik All suggestions were fixed. Thank you!

**Review by mrkulik — APPROVED** [review]
@ValentinaIancu-Gini Thank you!

**Pull request merged**